src/crystal_lib/parser.cr

```diff
@@ -39,6 +47,22 @@ class CrystalLib::Parser
     end
   end
 
+  def generate_comments(cursor)
+    if @options.includes?(Option::ImportFullComments)
+      cursor.raw_comment_text.try do |comment|
+        # Convert Doxygen comments
+        # (see http://www.doxygen.nl/manual/docblocks.html)
+        comment
+          .sub(/\/\*[\*!]\h*/m, "")       # Javadoc/Qt style comment block opening
```

Comment:
It's probably more performant to combine all search patterns into a single regular expression and calling `#sub` only once.

Reply:
Done!